backends/graphics/wincesdl/wincesdl-graphics.cpp

```diff
@@ -386,7 +386,6 @@ bool WINCESdlGraphicsManager::update_scalers() {
 
 		return true;
 	} else if (CEDevice::hasWideResolution()) {
-#ifdef USE_ARM_SCALER_ASM
```

Comment:
If this gets merged, the code inside the `#ifdef` needs to be removed as well

Reply:
I had kept this in because it doesn’t technically require the special assembly to function, though in either case it’s probably a moot point since this is the WinCE backend…